### PR TITLE
[467063] allow creating default methods from active annotations

### PR DESCRIPTION
**Pull request by cdietrich**
[467063] allow creating default methods from active annotations

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=467063

File: org.eclipse.xtend.core/src/org/eclipse/xtend/core/macro/declaration/JvmBasedDeclarations.xtend [hunk 1]
```diff
@@ -1052,6 +1052,10 @@ class MutableJvmMethodDeclarationImpl extends JvmMethodDeclarationImpl implement
 	override setDefault(boolean isDefault) {
 		checkMutable
 		delegate.setDefault(isDefault)
+		if (isDefault) {
+			delegate.abstract = false
```
Comment (szarnekow) [inline org.eclipse.xtend.core/src/org/eclipse/xtend/core/macro/declaration/JvmBasedDeclarations.xtend]: Shoudl we also set default to false when we set abstract to true?

**Comment by szarnekow** [comment]
Looks good.

File: org.eclipse.xtend.core/src/org/eclipse/xtend/core/macro/declaration/JvmBasedDeclarations.xtend (hunk 1 above)
Comment (cdietrich) [inline org.eclipse.xtend.core/src/org/eclipse/xtend/core/macro/declaration/JvmBasedDeclarations.xtend]: adapted (same for static)

**Pull request merged**